src/java/com/unifina/security/UserClassLoader.java

```diff
@@ -76,35 +63,7 @@ protected ModuleClassCollector(InnerLoader cl, CompilationUnit unit, SourceUnit
 			super(cl,unit,su);
 		}
     }
-    
```

Comment:
if no test breaks, it's probably dead and good to go (this looks kind of fundamental, so either it's useful and everything breaks; or not). But static analysis might not catch stuff that's called by
* groovy magic
* reflection through fully qualified class name stored in MySQL (modules)

Of course both of those could be cool to get rid of eventually...

Reply:
it was reported as unused by IntelliJ too. Unit tests pass on my machine.